### PR TITLE
Add new yale integration

### PR DESCRIPTION
**Pull request by bdraco**
## Proposed change

Full plan
https://github.com/home-assistant/home-assistant.io/pull/33890

## Type of change

- [x] Add a new logo or icon for a new core integration
- [ ] Add a missing icon or logo for an existing core integration
- [ ] Add a new logo or icon for a custom integration (custom component)
  - [ ] I've added a link to my custom integration repository in the PR description
  - [ ] I've opened up a PR for my custom integration on the [Home Assistant
    Python wheels repository](https://github.com/home-assistant/wheels-custom-integrations)
- [ ] Replace an existing icon or logo with a higher quality version
- [ ] Replace an existing icon or logo after a branding change
- [ ] Removing an icon or logo

## Additional information

- This PR fixes or closes issue: fixes #https://github.com/home-assistant/core/issues/124460
- Link to code base pull request: https://github.com/home-assistant/core/pull/124677
- Link to documentation pull request: https://github.com/home-assistant/home-assistant.io/pull/33890
- Link to integration documentation on our website: 
- Link to custom integration repository:

## Checklist

- [ ] The added/replaced image(s) are **PNG**
- [ ] Icon image size is 256x256px (`icon.png`)
- [ ] hDPI icon image size is 512x512px for  (`icon@2x.png`)
- [ ] Logo image size has min 128px, but max 256px, on the shortest side (`logo.png`)
- [ ] hDPI logo image size has min 256px, but max 512px, on the shortest side (`logo@2x.png`)

### CODE REVIEW

File: core_integrations/yale_home [hunk 1]
```diff
@@ -1 +1 @@
-../core_brands/yale
```
Comment (joostlek) [inline core_integrations/yale_home:1]: Shouldn't this be symlinked to the brands one as well?

Comment (bdraco) [reply core_integrations/yale_home:1]: Yes I think thats better

**Review by joostlek — APPROVED** [review]

File: core_integrations/yale_home (hunk 1 above)
Comment (bdraco) [reply core_integrations/yale_home]: ac9d5b4b8f75fe756a3bd13a471040d55d9ec776

**Comment by bdraco** [comment]
Thanks

**Pull request merged**